src/metrics.rs

```diff
@@ -2530,7 +2547,7 @@ impl GooseAttack {
             if !self.configuration.report_file.is_empty() {
                 if let Some(starting) = self.metrics.starting {
                     let second_since_start =
-                        (Utc::now().timestamp() - starting.timestamp()) as usize;
+                        (request_metric.request_time - starting.timestamp() as u64) as usize;
```

Comment:
Now that we have the actual time of the request on the `GooseRequestMetric` we can use that to calculate the second on the graph that the request belongs to. This gives us more accurate data and as a side-effect it will prevent those gaps.